src/components/Person.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './Person.scss';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  const partner = sex === 'f' ? 'husband' : 'wife';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+      <p className="Person__age">
+        {age && `I am ${age}`}
+      </p>
+      <p className="Person__partner">
```

Comment:
Don't render this tag if age is null
```suggestion
      {age && (
         <p className="Person__age">
           {`I am ${age}`}
         </p>
      )}
      <p className="Person__partner">
```